tutorials/acme-protocol-acme-clients.mdx

```diff
@@ -628,3 +629,16 @@ $ curl https://foo.internal --cacert $(step path)/certs/root_ca.crt
 
 Hello TLS!
 ```
+
+### Certify The Web
+
+Certify The Web is a popular ACME Certificate Manager for Windows. It provides a full UI for managing thousands of certificates and supports a wide range of built in deployment tasks (to key vaults etc) plus many DNS API providers. Commercial licensing and support is also available.
+
+To use with `step-ca`:
+- Add your CA root certificate to *Local Machine > Trusted Certificate Authorities* and your CA intermediate to *Local Machine > Interemediate Certification Authorities*. This will make your endpoint certificate (and the other ACME certificates you issue from your CA) trusted on this machine.
```

Comment:
```suggestion
- Add your CA root certificate to *Local Machine > Trusted Certificate Authorities* and your CA intermediate to *Local Machine > Intermediate Certification Authorities*. This will make your endpoint certificate (and the other ACME certificates you issue from your CA) trusted on this machine.
```